microservices-connector/helm/templates/deployment.yaml

```diff
@@ -73,3 +73,17 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       terminationGracePeriodSeconds: 10
+---
+apiVersion: v1
+kind: Service
```

Comment:
This is not the right way to define service in helm. Please take https://github.com/opea-project/GenAIInfra/blob/main/helm-charts/chatqna/templates/service.yaml as a reference